testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-databricks-http-hms/hive.properties

```diff
@@ -0,0 +1,7 @@
+connector.name=hive
```

Comment:
Which test using this catalog?

Reply:
TestHiveDatabricksUnityCompatibility uses this.